OptimoveSDK/optimove-sdk/build.gradle

```diff
@@ -45,6 +45,10 @@ dependencies {
     implementation 'com.android.volley:volley:1.2.1'
     implementation 'com.google.firebase:firebase-messaging:23.0.0'
```

Comment:
I think we should consider changing this to compileOnly to support BoM now. Thoughts?

Reply:
very true, in our sdk it is compileOnly, this i think comes from optipush